src/Sarif.Viewer.VisualStudio.Core/ErrorList/ErrorListService.cs

```diff
@@ -698,6 +703,61 @@ private int WriteRunToErrorList(Run run, string logFilePath, SarifLog sarifLog,
                 this.ShowFilteredSuppressionStateColumn();
             }
 
+            if (skipRemapping == false)
```

Comment:
>skipRemapping == false [](http://example.com/codeflow?start=16&length=22)

Can you use !skipRemapping or did you want the explicit "false"?

Reply:
can be !skipRemapping, some teams prefer == false to ! so had a bit of a habit, I'll use ! for this repo for consistency